### PR TITLE
fix md_mpx_format_adjuster.py script

### PR DESCRIPTION
**Pull request by dhiraj0911**
**What kind of change does this PR introduce?**

- Bugfix for the automated documentation GitHub action PUSH workflow.
- Standardizes package manager to `npm` for both `talawa-api` and `talawa-admin`.
- Adds a Python script (`md_mpx_format_adjuster.py`) to ensure Markdown files are MPX compatible.

**Issue Number:** #1559 

---

**Did you add tests for your changes?**

- No new tests added. The changes pertain to the GitHub Actions workflow and documentation updates.
- Manual testing was conducted to ensure functionality and MPX compatibility.

---

**If relevant, did you update the documentation?**

- Documentation updated to reflect the shift from npm to yarn and the integration of `script.py`.
- Details on workflow changes and MPX compatibility considerations with Docusaurus v3 included.

---

**Summary**

- Addresses issues with GitHub action for automated documentation generation.
- Resolves inconsistencies in package managers and non-compliance with Docusaurus v3's MPX standards.
- Includes transition to yarn and introduction of `script.py` for Typedoc-generated Markdown files.

---

**Does this PR introduce a breaking change?**

- No breaking changes introduced.
- Focuses on resolving workflow issues and enhancing compatibility, not affecting core software functionality.

---

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

- Yes, the contributing guidelines have been followed.

### CODE REVIEW

**Comment by palisadoes** [comment]
Please work with @noman2002 on this. He'll be taking over the review of this PR

**Comment by palisadoes** [comment]
@Nitya-Pasrija Please be aware of this PR

**Comment by palisadoes** [comment]
Please fix the conflicting file

**Comment by Nitya-Pasrija** [comment]
On it

File: INSTALLATION.md [hunk 1]
```diff
@@ -6,42 +6,46 @@ This document provides instructions on how to set up and start a running instanc
 
 <!-- TOC -->
 
+- [Talawa-API Installation](#talawa-api-installation)
```
Comment (palisadoes) [inline INSTALLATION.md:9]: This file must not be included in this PR

**Review by palisadoes — CHANGES_REQUESTED** [review]
1. See comment
2. Have you determined whether the API workflow is updating the Talawa-Docs repository?

**Comment by palisadoes** [comment]
Please fix the conflicting files

**Comment by palisadoes** [comment]
@dhiraj0911 

1. Please fix the conflicting files.
2. Have you tested this exporting to Talawa-Docs

**Comment by dhiraj0911** [comment]
I've changed destination_branch to `develop` to see changes but not appearing there as verify-workflow branch was not there

**Comment by palisadoes** [comment]
Please reach out to Noman on Slack to schedule a time over the weekend for both of you to troubleshoot this. He's very busy during the week. Both of you are in the same time zone so it will be easier to fix it quickly

**Comment by dhiraj0911** [comment]
Okay, thanks

**Comment by palisadoes** [comment]
@dhiraj0911

1. Please fix the conflicting files
2. Have you managed to contact @noman2002?
3. You still haven't reported confirmation that the workflow is copying files to `Talawa-Docs` successfully

**Comment by dhiraj0911** [comment]
Yes, I messaged him on slack waiting for his reply

**Comment by dhiraj0911** [comment]
Till now I would like to contribute to other issues, can you assign them to me?

**Comment by palisadoes** [comment]
> Till now I would like to contribute to other issues, can you assign them to me?

I've assigned the original issue to myself

**Pull request closed without merging**